index.js

```diff
@@ -85,8 +85,12 @@ var E, InvalidAstError = E = (function() {
 }());
 
 
-// errorsP :: {labels :: [Label], inFunc :: Boolean, inIter :: Boolean, inSwitch :: Boolean} -> Node -> [InvalidAstError]
+// errorsP :: {labels :: [Label], inFunc :: Boolean, inIter :: Boolean, inSwitch :: Boolean, ES6 :: Boolean } -> Node -> [InvalidAstError]
 function errorsP(state) {
+
+  function isReservedWord (e) { return state.ES6 ? esutils.keyword.isReservedWordES6(e) : esutils.keyword.isReservedWordES5(e); }
```

Comment:
`let`, `static`, `yield`, and all future reserved words are only reserved in strict mode: http://www.ecma-international.org/ecma-262/6.0/index.html#sec-keywords